airsonic-main/src/main/java/org/airsonic/player/service/PodcastService.java

```diff
@@ -662,6 +684,12 @@ private boolean isEpisodeDeleted(PodcastEpisode episode) {
         return episode == null || episode.getStatus() == PodcastStatus.DELETED;
     }
 
+    // include episode == null to avoid attempts to download episodes for deleted channels
```

Comment:
this check should likely be added to isEpisodeDeleted

Reply:
It is already part of isEpisodeDeleted:

```java
    private boolean isEpisodeDeleted(PodcastEpisode episode) {
        episode = podcastDao.getEpisode(episode.getId());
        return episode == null || episode.getStatus() == PodcastStatus.DELETED;
    }
```